components/Dialogs/README.md

```diff
@@ -223,11 +248,89 @@ MDCAlertControllerThemer.applyScheme(scheme, to: alertController)
 // Step 1: Import the DialogThemer extension
 #import "MaterialDialogs+DialogThemer.h"
 
-// Step 2: Create or get a alert scheme
-MDCAlertScheme *alertScheme = [[MDCAlertScheme alloc] init];
+// Step 2: Create or get a Material container scheme
+MDCContainerScheme *scheme = [[MDCContainerScheme alloc] init];
+
+// Step 3 (optional): Customize the default theme by providing custom color or typography schemes:
+scheme.colorScheme = myColorScheme
+scheme.typographyScheme = myTypographyScheme
+
+// Step 4: Use the Material alert themer to theme an MDCAlertController instance
+[alertController applyThemeWithScheme:scheme];
+
+// Step 4: Alternatively, Use Material dialog presentation themer to theme your UIViewController instance:
+[myDialogViewController.mdc_dialogPresentationController applyThemeWithScheme: scheme];
+
+```
+<!--</div>-->
+
+<!-- Extracted from docs/action-theming.md -->
+
+### Action Theming
+
+Actions in MDCAlertController have emphasis which afects their theming.
+High emphasis actions generate contained buttons, medium emphasis actions generate outlined buttons and low emphasis actions generate text buttons.
+
+<div class="article__asset article__asset--screenshot">
+  <img src="docs/assets/dialogButtons.png" alt="Dialogs Actions" width="320">
+</div>
+
+Make sure the DialogThemer extension is added to your project:
+
+```bash
+pod 'MaterialComponents/Dialogs+DialogThemer'
+```
+
+The MDCContainerScheme defines design parameters that you can use to theme your dialogs.  The scheme is designed to be instanciated and customized once and be reused by many components.
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+```swift
+  // Create or reuse a Container scheme
+  let scheme = MDCContainerScheme()
+
+  // Creating an Alert dialog
+  let alert = MDCAlertController(title: "Button Theming",
+                                 message: "Lorem ipsum dolor sit amet, sit consectetur adipiscing")
+
+  // Adding three actions with different emphasis, creating buttons with different themes.
+  alert.addAction(MDCAlertAction(title:"All Right", emphasis: .high, handler: handler))
```

Comment:
Do we recommend having 3-emphasis buttons in the same Alert or is this more of a technical demo? If it's only a technical demo, I think it will be helpful to call that out.  If it's recommended, perhaps we can use an example directly from the Guidelines article?